kea_exporter/base_exporter.py

```diff
@@ -258,6 +205,11 @@ def setup_dhcp4_metrics(self):
                     'context': 'classes'
                 },
             },
+
+            'v4-lease-reuses': {
+                'metric' : 'reused_leases',
```

Comment:
```suggestion
                'metric' : 'leases_reused_total',
```

This should be normalized according to https://prometheus.io/docs/practices/naming/